website/src/css/custom.css

```diff
@@ -44,13 +44,13 @@ html[data-theme='dark'] .header-github-link:before {
   border-top: 1px solid #f2eeeee8;
 }
 
-html[data-theme='dark'] {
-  --ifm-background-color: #202429;
+html[data-theme='dark']:root {
+  --ifm-background-color: #12161b;
```

Comment:
Why do we need the `:root` selector here?

Reply:
@asafkorem is right. Just use either html or :root

Reply:
Btw, you can merge with line 17.

Reply:
`:root` has higher specification level than `html`, so there might be something that overrides the `html` version.. 🤔

Reply:
Actually maybe `html` selector doesn't work with JSX, because JSX is JavaScript **XML**..

Reply:
I'm not 100% sure but I think an `html` selector is not directly relevant to JSX and agree with what you said above.

Reply:
My apologies for the extra commit - had a technical issue and pushed 2nd commit without removing commented code. Fixed in latest commit.